src/test/Fake.Core.UnitTests/Fake.DotNet.Xdt.fs

```diff
@@ -0,0 +1,75 @@
+module Fake.DotNet.XdtTests
+
+open System
+open System.IO
+open System.Text
+open Fake.DotNet
+open Fake.IO
+open Expecto
+
+module TestData =
+    let fileName file =
+        Path.Combine(__SOURCE_DIRECTORY__, "TestFiles", "Fake.DotNet.Xdt.Files", file)
+
+    let replaceNewLines text =
+        RegularExpressions.Regex.Replace(text, @"\r\n?|\n", Environment.NewLine)
+
+    let exists file =
+        File.Exists(fileName file)
+
+    let require file =
+        if not (exists file) then
+            invalidArg "file" (sprintf "Unable to read test data from %s"
+                                       (Path.GetFullPath(fileName file)))
+
+    let read file =
+        require file
+        File.ReadAllText(fileName file, Encoding.UTF8)
+        |> replaceNewLines
+
+    let copy source dest =
+        require source
+        File.Copy(fileName source, fileName dest, true)
+
+    let delete file =
+        File.Delete(fileName file)
+
+[<Tests>]
+let tests =
+    testSequenced <| testList "Fake.DotNet.Xdt.Tests" [
```

Comment:
Is it possible to just copy the test-files or create unique names to remove the need to have them running sequentially?

Reply:
I'm not entirely sure what you mean. Can you give an example? I copied the tests that were provided for the old `XDTHelper` and converted them to use Expecto.

Reply:
One challenge for two of the functions is that they modify a file in-place, so I'm not sure how to resolve that without adding another dependency like a virtual file system.

Reply:
I'm happy to make the change, but I'm not sure what exactly to do here. Do you have a specific suggestion?